.circleci/config.yml

```diff
@@ -10,6 +10,13 @@ references:
         `which python3` -m venv venv
         venv/bin/pip install -U setuptools pip wheel
         venv/bin/pip install -r requirements/base.txt
+        venv/bin/pip install -U coverage
```

Comment:
I wonder if a new requirements file, based on `base.txt`, could be used to include test dependencies. I see there is already a `requirements/local.txt` file that just pulls in `base.txt`. With this in mind, could there be a file `requirements/circleci.txt` (or use `requirements/local.txt`);

```
-r base.txt

coverage>=6.4.1
```

On the other hand, there may be a good reason to install `coverage` it explicitly here.